packages/core/android/src/main/kotlin/com/datadog/reactnative/NoOpViewTrackingStrategy.kt

```diff
@@ -7,13 +7,14 @@
 package com.datadog.reactnative
 
 import android.content.Context
+import com.datadog.android.api.SdkCore
 import com.datadog.android.rum.tracking.ViewTrackingStrategy
 
 /**
  * No-op implementation of the [ViewTrackingStrategy].
  */
 object NoOpViewTrackingStrategy : ViewTrackingStrategy {
```

Comment:
Isn't NoOpViewTrackingStrategy available in the SDK now?

Reply:
Unfortunately it's `internal`.

Reply:
maybe we could make it public 🤔